tests/data/README.md

```diff
@@ -411,6 +411,14 @@ Box structure of the items in this file:
 [alpha] [alpha]
 ```
 
+### File [alpha_noispe.avif](alpha_noispe.avif)
```

Comment:
Nit/optional: Since we are using an underscore in the file name, it would be clearer to use one more: alpha_no_ispe.avif

Reply:
This is for consistency with `color_grid_alpha_nogrid.avif` which is not written `color_grid_alpha_no_grid.avif`.

Reply:
I see. This is fine. (When I first saw "noispe", it looked like a misspelled "noise").